bigbluebutton-html5/imports/ui/components/logout-confirmation/component.jsx

```diff
@@ -35,12 +37,18 @@ class LeaveConfirmation extends Component {
     super(props);
 
     this.handleLeaveConfirmation = this.handleLeaveConfirmation.bind(this);
+    this.handleCancleLogout = this.handleCancleLogout.bind(this);
   }
 
   handleLeaveConfirmation() {
+    Singleton.clear();
```

Comment:
I still dont understand why you are clearing all the LocalStorage. If you really need, just clear the values you are using.